migrations/versions/rev-2020-11-21-21:05:50-4642459983c0_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
+
+Revision ID: 4642459983c0
+Revises: 5ce97283f4bd
+Create Date: 2020-11-21 21:05:50.950332
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '4642459983c0'
+down_revision = '5ce97283f4bd'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('orders', 'deleted_at')
```

Comment:
You need to delete all orders where deleted_at is set before dropping the deleted column otherwise all deleted orders will become undeleted